hydroshare/settings.py

```diff
@@ -543,6 +545,7 @@ def _(s): return s # noqa
 CRISPY_TEMPLATE_PACK = "bootstrap"
 
 DEFAULT_AUTHENTICATION_CLASSES = (
+    "hs_core.authentication.BasicOIDCAuthentication",
     "rest_framework.authentication.BasicAuthentication",
```

Comment:
This PR will supports both the mezzanine password and the keycloak password. We'll probably want to eventually phase out the mezzanine password, though I do not see any rush.

Reply:
Maybe create an issue so that we don't forget about phasing it out?